### DISCUSSION TITLE
recipes/sides/quinoa/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#927] [Comments]
# recipes/sides/quinoa/

A collection of my recipes.

https://nicholaswilde.io/recipes/sides/quinoa/

**Comment by nicholaswilde** [comment]
100 g uncooked= 300 g cooked

**Comment by nicholaswilde** [comment]
8 servings = 267 g uncooked quinoa

**Reply by nicholaswilde** [reply]
200 g uncooked quinoa = 6 servings

**Reply by nicholaswilde** [reply]
450 g water = 6 servings
600 g water = 8 servings
